### DISCUSSION TITLE
Tab Session Manager

### DISCUSSION THREAD

**Discussion by GHNewbiee** [Floorp-Projects/Floorp#1071] [Q&A]
Hi,

I have installed Tab Session Manager extension but, unfortunately, its icon is not displayed on the toolbar. Is there anything I can do?

*Tia*

**Comment by CutterKnife** [comment]
Click on the Extensions button on the toolbar, then click on the Settings button to the right of that add-on to see the option to pin it to the toolbar.

**Comment by GHNewbiee** [comment]
@CutterKnife *Talfyh* I do not see the option. Am I doing something wrong?

![floorp](https://github.com/Floorp-Projects/Floorp/assets/26875662/efa2b8f4-d3fb-4bb2-a2b9-2d56edc590e6)

**Reply by CutterKnife** [reply]
https://imgur.com/a/PGnkC8Q

**Comment by GHNewbiee** [comment]
There is not such a button on the toolbar.

![floorp](https://github.com/Floorp-Projects/Floorp/assets/26875662/4cbe9c6a-11eb-4e1b-b692-a07fc50fdb67)

**Reply by CutterKnife** [reply]
Go to the "Customize Toolbar" page ([the same way as in Firefox](https://support.mozilla.org/en-US/kb/customize-firefox-controls-buttons-and-toolbars)) and enable the "Unified Extensions Button".

**Reply by GHNewbiee** [reply]
It's done! *Tal*
